### PR TITLE
Microverse templete 1 project

### PR DESCRIPTION
**Pull request by Rustamxon7**
I created the first page of template 1. README file changed and linter file added. All code is written in HTML&CSS only.

### CODE REVIEW

**Comment by Rustamxon7** [comment]
I created **Template 1** mobile version of the homepage. Please let me know if there are any problems.

File: index.html [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+        <h3>LET'S CONNECT</h3>
+        <div class="icons">
+          <img src="images/Twitter.svg" alt="twitter">
+          <img src="images/Linkedin-icon.svg" alt="linkedin">
+          <img src="images/Medium.svg" alt="medium">
+          <img src="images/GitHub.svg" alt="github">
+          <img src="images/Vector.svg" alt="vector">
+
+        </div>
+      </div>
```
Comment (arslanbisharat) [inline index.html:41]: Please make sure you have ul instead of ddiv tag

File: index.html [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+        <img src="images/Union.svg" alt="" class="icon">
+      </nav>
+    </header>
+    <div class="headline">
+      <div class="maintext">
+        <h1>I’m Rustamjon Glad to see you!</h1>
+        <h2>
+          I’m a software developer! I can help you build a product , feature or
+          website Look through some of my work and experience! If you like what
+          you see and have a project you need coded, don’t hestiate to contact me.
```
Comment (arslanbisharat) [inline index.html:30]: It should be enclosed in p taag

**Review by arslanbisharat — CHANGES_REQUESTED** [review]
# Changes Requested, 

Hi @Rustamxon7, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: index.html (hunk 1 above)
Comment (Rustamxon7) [reply index.html]: I changed div to ul.

File: index.html (hunk 2 above)
Comment (Rustamxon7) [reply index.html]: That one has also done.

**Comment by Rustamxon7** [comment]
I 

> # Changes Requested,
> Hi @Rustamxon7,
> 
> Good job so far!
> There are some issues that you still need to work on to go to the next project but you are almost there!
> 
> ## Required Changes ♻️
> _Check the comments under the review._
> 
> ## Optional suggestions
> _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
> Cheers and Happy coding!👏👏👏
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
> 
> _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Did I really complete it?

**Pull request merged**